packages/adp-tooling/templates/project/package.json

```diff
@@ -15,14 +15,15 @@
         "@sap-ux/ui5-proxy-middleware": "^1.3.0",
         "@sap-ux/deploy-tooling": "^0.11.7"<%}%>,
         "@ui5/task-adaptation": "^1.3.0",
-        "@ui5/cli": "^3.9.2"
+        "@ui5/cli": "^3.9.2"<%if (locals.deploy) {%>,
+        "rimraf": "^5.0.5"<%}%>
     },
     "scripts": {
         "build": "ui5 build --exclude-task generateFlexChangesBundle generateComponentPreload minify --clean-dest",
         "start": "<%= locals.options?.fioriTools ? 'fiori run' : 'ui5 serve' %> --open /test/flp.html#app-preview",
         "start-editor": "<%= locals.options?.fioriTools ? 'fiori run' : 'ui5 serve' %> --open /test/adaptation-editor.html"<%if (locals.deploy) {%>,
-        "deploy": "ui5 build --config ui5-deploy.yaml --exclude-task <%= locals.options?.fioriTools ? 'deploy-to-abap' : 'abap-deploy-task' %> generateFlexChangesBundle generateComponentPreload --clean-dest && <%= locals.options?.fioriTools ? 'fiori deploy' : 'deploy' %> --config ui5-deploy.yaml",
-        "undeploy": "<%= locals.options?.fioriTools ? 'fiori undeploy' : 'undeploy' %> --config ui5-deploy.yaml --lrep \"apps/<%= app.reference %>/appVariants/<%= app.id %>/\"",
+        "deploy": "npm run build && <%= locals.options?.fioriTools ? 'fiori deploy' : 'deploy' %> --config ui5-deploy.yaml && rimraf archive.zip",
```

Comment:
This is quite a change. Before your change, the the build configuration from the `ui5-deploy.yaml` was used for building the sources, now it is using the `ui5.yaml`. Are you sure about that?

So, maybe something like `npm run build -- --config ui5-deploy.yaml && <%= locals.options?.fioriTools ? 'fiori deploy' : 'deploy' %> --config ui5-deploy.yaml && rimraf archive.zip` is the better option.

Reply:
Well we don't need ui5-deploy.yaml for build, we need it only for deployment.

Reply:
Actually, you do. The idea of having a separate `ui5.yaml` and a `ui5-deploy.yaml` is that you can have separate build configurations e.g. to filter test/local files for the deploy (productive) build. So, having the clean separation here makes sense and should not be removed even if the generated (and not modified) files are the same.

Reply:
I tested your proposal adding the ui5-deploy.yaml for build and deploy is failing there is some issue in the zipped archive of the project which is catched by the backend. Our deployment for cloud project is not working at the moment and we have very high ticket for it. Do you think we could do a sync to discuss and agree on a solution to expedite the resolution of the problem?